core/domain/exp_services.py

```diff
@@ -1684,15 +1683,14 @@ def get_scaled_average_rating(ratings):
         return 0
     average_rating = get_average_rating(ratings)
     z = 1.9599639715843482
-    x = python_utils.divide((average_rating - 1), 4)
+    x = (average_rating - 1) / 4
     # The following calculates the lower bound Wilson Score as documented
     # http://www.goproblems.com/test/wilson/wilson.php?v1=0&v2=0&v3=0&v4=&v5=1
-    a = x + python_utils.divide((z**2), (2 * n))
+    a = x + ((z**2) / (2 * n))
     b = z * math.sqrt(
-        python_utils.divide((x * (1 - x)), n) + python_utils.divide(
-            (z**2), (4 * n**2)))
-    wilson_score_lower_bound = python_utils.divide(
-        (a - b), (1 + python_utils.divide(z**2, n)))
+        ((x * (1 - x)) / n) + ((z**2) / (4 * n**2)))
```

Comment:
Can be moved to previous line

Reply:
@anurag629 I think you can also remove the outermost parens when doing that.

Reply:
I think the parens should be kept 
```
math.sqrt(
    (
        (
            x * 
            (1 - x)
        ) / 
        n
    ) + 
    (
        (z**2) / 
        (4 * n**2)
    )
)
```

I know that the priority is on `/` but I would keep the parens to make the priority explicit.